tools/scylla-nodetool.cc

```diff
@@ -3742,6 +3978,140 @@ For more information, see: {}"
                 table_stats_operation
             }
         },
+        {
+            {
+                "tasks",
+                "Manages task manager tasks",
+                "",
+                { },
+                {
+                    typed_option<sstring>("command", "The uuid of a task", 1),
+                },
+                {
+                        {
+                            "abort",
+                            "Aborts the task",
+                            fmt::format(R"(
+                            For more information, see: {}"
+                            )", doc_link("operating-scylla/nodetool-commands/tasks/abort.html")),
+                            { },
+                            {
+                                typed_option<sstring>("id", "The uuid of a task", 1),
+                            },
+                        },
+                        {
+                            "list",
+                            "Gets a list of tasks in a given module",
+            fmt::format(R"(
+            For more information, see: {}"
+            )", doc_link("operating-scylla/nodetool-commands/tasks/list.html")),
+                            {
+                                typed_option<>("internal", "Show internal tasks"),
+                                typed_option<sstring>("keyspace,ks", "The keyspace name"),
+                                typed_option<sstring>("table,t", "The table name"),
+                                typed_option<int>("period", 10, "The number of seconds after which the list will be updated"),
```

Comment:
Re-print the task list after period seconds. Can be used to monitor the task-list for extended period of time.